solr/solrj/src/java/org/apache/solr/client/solrj/impl/BaseHttpSolrClient.java

```diff
@@ -62,6 +63,9 @@ public static RemoteExecutionException create(String host, NamedList errResponse
       if (errObj != null) {
         Number code = (Number) getObjectByPath(errObj, true, Collections.singletonList("code"));
         String msg = (String) getObjectByPath(errObj, true, Collections.singletonList("msg"));
+        if(msg == null) msg = "";
```

Comment:
There's already a null check later, what kinds of messages do we get here that are useful and don't leak too much internals?